locale/en-us.yaml

```diff
@@ -1068,6 +1068,27 @@ language_name_unknown:
   other: Unknown
 language_unknown_options:
   other: Unknown options
+sample_yaml_perl:
```

Comment:
What about Python?

Reply:
I left it out due to the story points and the Perl specific title and description. It's added now.